solr_to_es/solrSource.py

```diff
@@ -93,10 +103,13 @@ def __next__(self):
         if self.docs is None:
             if self.current * self.rows < self.max:
                 self.current += 1
-                response = self.solr_conn.search(self.query, rows=self.rows,
-                                                 start=(self.current - 1) * self.rows,
-                                                 **self.options)
-                self.docs = iter(response.docs)
+                url = self.solr_url+ '/select?q=' + self.query + '&rows=' + str(self.rows) + "&start=" + str(((self.current - 1) * self.rows))
+                if self.fl:
+                    url = url + "&fl=" + (','.join(self.fl))
+                print(url)
```

Comment:
Should this still be here?

Reply:
Normally yes, any reason why we should remove building the URL to request?

Reply:
I think I just meant the `print(url)`

Reply:
Sorry misunderstood that, `print(url)` has been removed in the last commit